src/nft/components/collection/MarketplaceSelect.tsx

```diff
@@ -34,12 +44,19 @@ export const MARKETPLACE_ITEMS = {
   sudoswap: 'SudoSwap',
 }
 
+function getMarketLogoSrc(market: string) {
+  const marketplaceItem = Object.keys(MARKETPLACE_ITEMS).find(
+    (key) => MARKETPLACE_ITEMS[key as keyof typeof MARKETPLACE_ITEMS] === market
+  )
+  return `/nft/svgs/marketplaces/${marketplaceItem}.svg`
+}
+
 export const FilterItem = ({
   title,
   element,
   onClick,
 }: {
-  title: string
+  title: string | JSX.Element
```

Comment:
ReactNode should work

Reply:
Is there a reason for using one of the other?